### DISCUSSION TITLE
Kohya_ss X/Y/Z Plot

### DISCUSSION THREAD

**Discussion by rubikstriangle** [bmaltais/kohya_ss#1998] [Ideas]
I've been creating a lot of LoRa.  I always create an X/Y/Z plot in automatic1111 of the epochs on the X axis and six seed variations on the Y so that I can determine which epoch was the most successful, the seed helps identify which epoch is consistently the best.   I imagine that this is a pretty typical workflow.  It would save so much time and be so much better organized if an xyz plot could be generated strait after the  LoRa is created and saved in the same folder as the LoRA with an easily identifiable fine name.  

I had a look at automatic1111's xyz_grid.py script and tinkered with it a bit, but it is beyond my skill set to script this one up, however it would be far better if it was tightly integrated into Kohya_ss.  Just throwing it out there to see if the idea sticks....

**Comment by 5KilosOfCheese** [comment]
Can't you just write the variations out to the sampling field and have it generate the list every epoch? Just make every prompt on separate lines. That is how XYZ scripts work to begin with. To make managing it easier you could do it in Excel or similar spreadsheet via simple string functions. You won't get the graph, but it isn't like that is hard to make with a basic photoshop or just drag and drop to paint - or a basic python script from somewhere.

Personally I don't test the LORAs in the GUI, for the simple reason that to actually test it I need to see how it works in my style of prompting and workflow.
